pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -1146,6 +1159,20 @@ func (c InstallerController) Sync(ctx context.Context, syncCtx factory.SyncConte
 	} else if updatedNodeReportOnSuccessfulUpdateFn != nil {
 		updatedNodeReportOnSuccessfulUpdateFn()
 	}
+
+	// Perform a live get to obtain the RV of the object we just applied as for
```

Comment:
If we get a non-nil error from ApplyStaticPodOperatorStatus, isn't it possible that the apply succeeded but our client observed an error (like an I/O error) instead of reading a successful HTTP response?

Reply:
It is not possible since the Apply request is done last: 
https://github.com/openshift/library-go/blob/master/pkg/operator/genericoperatorclient/dynamic_operator_client.go#L273-L280
https://github.com/openshift/library-go/blob/master/pkg/operator/genericoperatorclient/dynamic_operator_client.go#L371-L379